doc/source/whatsnew/v0.24.0.txt

```diff
@@ -441,7 +441,7 @@ Datetimelike
 Timedelta
 ^^^^^^^^^
 
--
+- Fixed bug where array of timestamp and deltas raised a TypeError: unsupported operand type(s) for -: ``numpy.ndarray`` and ``Timedelta`` (:issue:`21980`)
```

Comment:
timestamp --> :class:`Timestamp`
double back ticks around ``TypeError``
don't need the full error message

Can you clarify what "array of timestamp and deltas" means?  e.g.

```
Bug where subtracting :class:`Timedelta` from an object-dtyped array would raise ``TypeError``
```